pkgs/development/python-modules/tinygrad/fix-dlopen-cuda.patch

```diff
@@ -3,8 +3,8 @@ index a30c8f53..e2078ff6 100644
 --- a/tinygrad/runtime/autogen/cuda.py
 +++ b/tinygrad/runtime/autogen/cuda.py
 @@ -145,7 +145,19 @@ def char_pointer_cast(string, encoding='utf-8'):
- 
```

Comment:
Did you strip the final space here? I think that is important.

You can ignore whitespace removal with git in attributes for patch and diff files.

Reply:
I removed them.
My editor is automatically trimming trailing whitespaces.